commands/enrich/enrichgraph/enrichgraph.go

```diff
@@ -0,0 +1,23 @@
+package enrichgraph
+
+import (
+	"github.com/jfrog/jfrog-client-go/xray"
+	"github.com/jfrog/jfrog-client-go/xray/services"
+)
+
+const (
+	EnrichMinimumVersionXray = "3.90"
```

Comment:
```suggestion
	EnrichMinimumVersionXray = "3.101.3"
```
As mentioned, set the actual release version for this feature

Reply:
fixed